static/js/NavSidebar.jsx

```diff
@@ -267,19 +271,21 @@ const getSidebarFooterData = () => [{'he': 'אודות','en': 'About', 'url': 'w
 
 const SidebarFooter = () => {
 
-  data = getSidebarFooterData();
+  const data = getSidebarFooterData();
 
   return (
-    <Module>
-      <h3/>
-      <div className="footerContainer">
-        {data.map(footerLink => 
-          <a href={footerLink.url}>
-            <InterfaceText text={{'en': footerLink.en, 'he': footerLink.he}}  />
-          </a>
-        )}
-      </div>
-    </Module>
+    <div className = "stickySidebarFooter">
+      <Module>
```

Comment:
I dont think this should be wrapped as a <Module> anymore

Reply:
Can you clarify? It might be by mistake the tag you were referencing got deleted from the message. Thanks!